### PR TITLE
[Admin] Support for CMF routes

### PR DESCRIPTION
**Pull request by pamil**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Related tickets | based on #5409 
| License         | MIT

### CODE REVIEW

**Comment by dantleech** [comment]
Hey, just thought I would check: have you considered using routing auto component as an alternative or in addition to manually managing routes?

I am currently building a CMS only Symfony CMF and the Sylius Admin and Resource bundles (I am actually giving a workshop on this at web summer camp in September). It would be great to focus our efforts together if it makes sense.

**Comment by pamil** [comment]
I'm just porting the existing functionality from old admin panel to the new one now. Anyway, these routes are only used for static content right now and probably we won't change much in routing because the long-awaited beta is coming :tada:

As far as I know we won't be focusing on built-in content management much in Sylius (e-commerce is enough. @pjedrzejewski will certainly know more about the plans! :)

**Comment by dantleech** [comment]
ok, just wanted to make sure, without going to deep, that we wern't duplicating work :)

File: src/Sylius/Behat/Context/Ui/Admin/ManagingStaticContentsContext.php [hunk 1]
```diff
@@ -19,7 +19,7 @@
 use Webmozart\Assert\Assert;
 
 /**
- * @author Łukasz Chruściel <lukasz.chrusciel@lakion.com>
+ * @author Kamil Kokot <kamil.kokot@lakion.com>
```
Comment (lchrusciel) [inline src/Sylius/Behat/Context/Ui/Admin/ManagingStaticContentsContext.php]: :(

Comment (pamil) [inline src/Sylius/Behat/Context/Ui/Admin/ManagingStaticContentsContext.php]: That was too good code indeed to give it up to someone else! :tada:

File: src/Sylius/Bundle/CoreBundle/Fixture/Factory/RouteExampleFactory.php [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+        $this->faker = \Faker\Factory::create();
+        $this->optionsResolver =
+            (new OptionsResolver())
+                ->setDefault('name', function (Options $options) {
+                    return StringInflector::nameToCode($this->faker->words(3, true));
+                })
+                ->setDefault('content', LazyOption::randomOne($staticContentRepository))
+                ->setAllowedTypes('content', ['string', StaticContent::class])
+                ->setNormalizer('content', LazyOption::findOneBy($staticContentRepository, 'name'))
```
Comment (lchrusciel) [inline src/Sylius/Bundle/CoreBundle/Fixture/Factory/RouteExampleFactory.php]: last bracket should go to the new line

**Comment by pjedrzejewski** [comment]
Thank you Kamil!

**Pull request merged**